eng/Versions.props

```diff
@@ -58,11 +58,11 @@
     <MicrosoftNETHostModelVersion>8.0.0-rtm.23504.8</MicrosoftNETHostModelVersion>
     <MicrosoftExtensionsFileSystemGlobbingPackageVersion>8.0.0-rtm.23504.8</MicrosoftExtensionsFileSystemGlobbingPackageVersion>
     <MicrosoftExtensionsDependencyModelVersion>$(MicrosoftExtensionsDependencyModelPackageVersion)</MicrosoftExtensionsDependencyModelVersion>
+    <MicrosoftExtensionsLoggingVersion>$(MicrosoftExtensionsLoggingConsoleVersion)</MicrosoftExtensionsLoggingVersion>
+    <MicrosoftExtensionsLoggingAbstractionsVersion>$(MicrosoftExtensionsLoggingConsoleVersion)</MicrosoftExtensionsLoggingAbstractionsVersion>
     <MicrosoftExtensionsLoggingConsoleVersion>8.0.0-rtm.23504.8</MicrosoftExtensionsLoggingConsoleVersion>
```

Comment:
Would this need to be moved up, above the two new properties?

Reply:
Any reason to not just use the static version as in https://github.com/dotnet/sdk/pull/35760? Dependency flow would update the properties anyway.

Reply:
The conflict resolution really messed things up from a simple cherry-pick.